test/NonFungibleCollectable.test.js

```diff
@@ -75,8 +83,8 @@ contract('NonFungibleToken', accounts => {
 
     describe('when the user tries to claim a token that was not assigned to them via a valid coupon', () => {
       it('reverts', async () => {
-        const {hash, v, r, s} = signatureObject
-        await assertRevert(token.claimCollectableWithCoupon(_secondTokenId, hash, v, r, s, {from: recipientAddress}))
+        const {prefixedHash, v, r, s} = signatureObject
+        await assertRevert(token.claimCollectableWithCoupon(_secondTokenId, prefixedHash, v, r, s, {from: recipientAddress}))
         const newOwner = await token.ownerOf(_firstTokenId)
```

Comment:
I think this should be `const newOwner = await token.ownerOf(_secondTokenId)` in order to agree with trying to claim the token that has not been assigned by a valid coupon. Changing it still gets the expected `revert()`.

Reply:
Yes, agreed, but since we are using the signature for the 1st token, it makes sense to test both tokens actually